src/main/webapp/app/view/project/admin/users/project-users.component.html

```diff
@@ -1,5 +1,61 @@
 <div>
-    <h3 id="page-heading">
-        <span jhiTranslate="eventPlannerApp.project.admin.navbar.users">Users</span>
-    </h3>
+  <h3 id="page-heading">
+    <span jhiTranslate="eventPlannerApp.project.admin.navbar.users"></span>
+
+    <div class="float-right">
+      <app-clientside-filter id="users-clientside-filter" class="float-left"
+                             (valueChange)="filterData($event)"></app-clientside-filter>
+
+      <button id="jh-create-entity"
+              class="btn btn-primary jh-create-entity invite-user"
+              [routerLink]="['', {outlets: {modal: ['projects', projectId, 'users', 'invite']}}]">
+        <fa-icon icon="plus"></fa-icon>
+        <span jhiTranslate="userManagement.invite"></span>
```

Comment:
interesting, that this doesn't have the `eventPlannerApp` prefix.. did you create a new file?

Reply:
Nein alle User-bezogenen Übersetzungen waren schon dort drin. Aber so ganz konsistent ist's eh nicht

Reply:
aha, weil es teil des ADMIN bereichts ist.. ich sehs.. ok ;)